### ISSUE TITLE
Monitoring: Missing suppressions interface

### ISSUE THREAD

**Issue by enver** [rackspace/pyrax#582]
There is no interface in cloud monitoring for suppressions. Any ETA on implementation?

**Comment by briancurtin** [comment]
There are no plans for this. We're working on a replacement for Pyrax in the [OpenStack SDK](https://pypi.python.org/pypi/openstacksdk) and accompanying [Rackspace SDK](https://pypi.python.org/pypi/rackspacesdk). We don't have any support for Cloud Monitoring in the Rackspace extension yet, but that's where this type of code will end up.

**Comment by linaksa** [comment]
Hi @briancurtin Are OpenStack SDK and Rackspace SDK ready to use in production?

**Comment by briancurtin** [comment]
Neither of them are complete enough to really be considered production ready in terms of the APIs that are available, but what is out there now is generally quite stable. The Rackspace extension to OpenStack SDK really only just supplies authentication and one tiny extension at the moment -- that will eventually contain support for all of our own services, like Cloud Monitoring.

**Comment by linaksa** [comment]
Thank you @briancurtin Per https://developer.rackspace.com/sdks/python/ the Rackspace's official SDK for Python is pyrax. Would you know any rough estimate (3 months, 6 months, etc.) when Rackspace SDK will take over pyrax?

**Comment by briancurtin** [comment]
Yeah, I need to do some writing on what we're doing here now that things are a little more solidified, but also still a bit malleable. OpenStack SDK, the basis for the Rackspace SDK, is probably more on a 3 month schedule (should be quicker in reality, we're just working on a few key low level pieces and then a lot of functional/live testing).

Getting the Rackspace extension up to feature parity with Pyrax is more of a 6 month thing, roughly. I haven't had time to really estimate all of the work that will have to go into the Rackspace specific parts, but off the top of my head it'll be some time early next year for us to have a fully tested (including functional/live tests) Rackspace offering that is production ready and able to replace what Pyrax currently offers.

**Comment by linaksa** [comment]
Awesome, thanks again for quick response @briancurtin

**Comment by enver** [comment]
Please review and merge pull request https://github.com/rackspace/pyrax/pull/585

**Comment by enver** [comment]
Please review and merge pull request https://github.com/rackspace/pyrax/pull/586

**Comment by enver** [comment]
Ok, so pull request is not accepted. If anyone else needs this you can get it from https://github.com/enver/pyrax (see installation instructions from github on https://github.com/rackspace/pyrax#installation)
